blueprints/showbie-tooling/files/circle.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
         - pip install --upgrade pip
     override:
         - pip install pre-commit
-        - yarn global add bower eslint-plugin-ember-suave
+        - yarn global add bower
```

Comment:
Might need to add `eslint-plugin-ember` here for CI to run properly.

Reply:
I've been running eslint without having the plugins or configs installed globally without issue. Looks like the CI catches the Ember rules since one of them isn't passing...